dietpi/dietpi-software

```diff
@@ -4199,6 +4205,111 @@ _EOF_
 			G_EXEC sysctl -w net.core.rmem_max=2500000
 		fi
 
+		if To_Install 206 # microblog.pub
+		then
+			## 1. Create a python environment py3.10 via pyenc
+			##	  see: https://python-poetry.org/docs/managing-environments/
+			local micro_pyenv_dir="/home/dietpi/.pyenv_microblog"
```

Comment:
Please do not use the `dietpi` user, which is about to be removed. System services shall run as their dedicated service user.
```suggestion
			local micro_pyenv_dir="/mnt/dietpi_userdata/microblog-pub/.pyenv"
```
If it is possible to split install and data/config directories:
```suggestion
			local micro_pyenv_dir="/opt/microblog-pub/.pyenv"
```
and config/data to `/mnt/dietpi_userdata/microblog-pub` instead.